doc/release/technotes/README.extern.rst

```diff
@@ -85,11 +86,13 @@ To help with mapping types between C and Chapel, Chapel installations
 contain a standard module named 'SysCTypes' (located under
 $CHPL_HOME/modules/standard/gen/...).  This module defines a number of
```

Comment:
I want to make more substantial changes to this file (including fixing such cross-references) but want to make that Part 2 in another pull request.